src/components/atoms/dropdown/modal.tsx

```diff
@@ -7,6 +7,7 @@ import {
   ScrollView,
   View,
   Text,
+  TouchableHighlight,
```

Comment:
`react-native-gesture-handler`'s TouchableHighlight responds faster that React Native's own in most cases in my testing but here it simply doesn't work, it breaks the onPress on Android, so using the react-native built in one here.

Reply:
Why are we using two `TouchableHighlights`? How bad is the performance of the RN built in one? It seems it would be better to have a single component throughout the app rather than bringing in a new library/dependency and using it in some places but not others.

Reply:
For cards, in my testing it doesn't show the pressed state on normal presses until the screen transition starts, which makes it fairly useless. I'm intending to look a little more into the internals (I suspect react native's requires a state update while gesture handler's doesn't).

But for now this is just a low priority trial, the request from Antoine only came through today and we haven't validated that we even definitely want this style of highlighting. I've also set this to draft to ensure no misunderstandings.

Reply:
https://reactnative.dev/docs/pressable